src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,17 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class ParserTest {
+    @Test
+    public void parseWrongCommandTest() {
```

Comment:
```suggestion
    public void parse_wrongCommand_exceptionThrown() {
```
Perhaps you can use this naming convention to better indicate the function you are testing!

Reply:
Thanks for pointing out the test method naming convention :)